systest/Makefile

```diff
@@ -300,7 +311,7 @@ tempest_11.6.1_undercloud_vlan:
 	export EXCLUDE_FILE=$@.yaml ;\
 	$(MAKE) -C . run_tests
 
-# Tempest Tests for 12.1.1 undercloud VE deployment
+# Tempest Tests for 12.1.x undercloud VE deployment
 tempest_12.1.1_undercloud_vxlan:
```

Comment:
Think about removing the 12.1.1 targets. We can parameterize the bigip version, as is done in the agent, so we aren't bound to having these ultra-specific target names.